wp-content/themes/orient-theme/comments.php

```diff
@@ -72,7 +72,7 @@
 		    '" size="30"' . $aria_req . ' /></p>'
 	),
 	'comment_notes_before' => '<p><strong>Any comments that do not follow the policy will not be published.</strong></p>',
-	'comment_notes_after' => '',
+	'comment_notes_after' => '<p id="wordcount">0 words</p>',
```

Comment:
The default value here should maybe be `0/200 words` to match what the Javascript will change it to, which is `count + "/200 words"`.